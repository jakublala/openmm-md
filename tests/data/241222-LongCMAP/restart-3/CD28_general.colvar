#! FIELDS time cmap d metad.bias uwall_b.bias uwall_idr.bias
 1.000000 175.090350 1.281246 52.253825 0.000000 0.000000
 2.000000 173.623958 1.292970 51.097248 0.000000 0.000000
 3.000000 172.193705 1.305521 53.502409 0.000000 0.000000
 4.000000 173.251590 1.316306 54.483455 0.000000 0.000000
 5.000000 172.792758 1.297444 49.682316 0.000000 0.000000
 6.000000 171.414497 1.315961 53.801698 0.000000 0.000000
 7.000000 174.439608 1.273341 51.005227 0.000000 0.000000
 8.000000 170.995567 1.304038 54.650390 0.000000 0.000000
 9.000000 174.281195 1.308882 51.952990 0.000000 0.000000
 10.000000 170.015758 1.325771 53.215243 0.000000 0.000000
 11.000000 171.638756 1.311754 53.241295 0.000000 0.000000
 12.000000 172.489851 1.303488 53.186324 0.000000 0.000000
 13.000000 172.828209 1.310648 51.328390 0.000000 0.000000
 14.000000 176.619076 1.287147 47.008198 0.000000 0.000000
 15.000000 171.294460 1.305779 54.077451 0.000000 0.000000
 16.000000 172.381891 1.307023 54.556683 0.000000 0.000000
 17.000000 170.348193 1.332407 52.728148 0.000000 0.000000
 18.000000 173.143896 1.321056 54.911960 0.000000 0.000000
 19.000000 171.419057 1.304629 54.830929 0.000000 0.000000
 20.000000 169.129021 1.324781 52.647876 0.000000 0.000000
 21.000000 169.740768 1.313769 50.785059 0.000000 0.000000
 22.000000 170.879262 1.314184 53.887198 0.000000 0.000000
 23.000000 170.144058 1.326750 52.212803 0.000000 0.000000
 24.000000 171.662030 1.319470 54.497512 0.000000 0.000000
 25.000000 170.216301 1.319540 52.298507 0.000000 0.000000
 26.000000 168.928083 1.313532 53.635314 0.000000 0.000000
 27.000000 169.745807 1.319497 52.138023 0.000000 0.000000
 28.000000 171.295202 1.320201 56.244404 0.000000 0.000000
 29.000000 169.492624 1.320228 52.421738 0.000000 0.000000
 30.000000 169.960362 1.324681 55.819111 0.000000 0.000000
 31.000000 174.490093 1.291007 52.834052 0.000000 0.000000
 32.000000 172.794412 1.318493 52.003926 0.000000 0.000000
 33.000000 174.457820 1.294865 53.866276 0.000000 0.000000
 34.000000 176.673293 1.274880 47.584287 0.000000 0.000000
 35.000000 177.140062 1.300636 49.007894 0.000000 0.000000
 36.000000 171.872815 1.302570 53.723514 0.000000 0.000000
 37.000000 170.523610 1.317119 53.388078 0.000000 0.000000
 38.000000 169.687050 1.320041 52.434511 0.000000 0.000000
 39.000000 171.935435 1.310392 55.091608 0.000000 0.000000
 40.000000 170.357898 1.300528 52.912713 0.000000 0.000000
 41.000000 168.667748 1.309535 54.186729 0.000000 0.000000
 42.000000 170.173167 1.312207 53.349888 0.000000 0.000000
 43.000000 169.511363 1.307997 52.502008 0.000000 0.000000
 44.000000 170.732089 1.301176 51.982060 0.000000 0.000000
 45.000000 169.834738 1.314768 55.693141 0.000000 0.000000
 46.000000 170.366519 1.292074 53.559716 0.000000 0.000000
 47.000000 170.866980 1.328772 55.882654 0.000000 0.000000
 48.000000 173.845122 1.301438 51.848121 0.000000 0.000000
 49.000000 173.057131 1.304890 54.802999 0.000000 0.000000
 50.000000 170.759789 1.317746 54.572788 0.000000 0.000000
 51.000000 171.548473 1.303220 55.664286 0.000000 0.000000
 52.000000 169.541252 1.305530 53.001285 0.000000 0.000000
 53.000000 170.600191 1.324231 55.593991 0.000000 0.000000
 54.000000 171.225488 1.314182 56.862110 0.000000 0.000000
 55.000000 169.717852 1.306326 53.950879 0.000000 0.000000
 56.000000 169.266436 1.319558 53.386164 0.000000 0.000000
 57.000000 169.208445 1.322569 54.115565 0.000000 0.000000
 58.000000 170.106555 1.305446 54.393518 0.000000 0.000000
 59.000000 169.895131 1.305174 57.109757 0.000000 0.000000
 60.000000 169.830740 1.305119 57.228695 0.000000 0.000000
 61.000000 172.365102 1.302952 55.142874 0.000000 0.000000
 62.000000 170.146792 1.302662 54.951568 0.000000 0.000000
 63.000000 168.209760 1.314518 54.956752 0.000000 0.000000
 64.000000 168.717456 1.299901 53.980558 0.000000 0.000000
 65.000000 170.719742 1.295833 54.246560 0.000000 0.000000
 66.000000 166.944471 1.309674 55.682816 0.000000 0.000000
 67.000000 167.752239 1.308282 55.844550 0.000000 0.000000
 68.000000 169.261404 1.310741 54.181486 0.000000 0.000000
 69.000000 168.546293 1.286966 53.355084 0.000000 0.000000
 70.000000 168.326907 1.309783 55.456636 0.000000 0.000000
 71.000000 169.139992 1.308444 54.322971 0.000000 0.000000
 72.000000 165.718662 1.315537 61.498611 0.000000 0.000000
 73.000000 167.779742 1.321527 57.863839 0.000000 0.000000
 74.000000 168.276345 1.306366 55.837984 0.000000 0.000000
 75.000000 166.929999 1.315337 56.993708 0.000000 0.000000
 76.000000 168.607937 1.300232 55.722268 0.000000 0.000000
 77.000000 169.184974 1.312185 55.495678 0.000000 0.000000
 78.000000 169.770062 1.289172 55.517972 0.000000 0.000000
 79.000000 168.950893 1.311399 55.523651 0.000000 0.000000
 80.000000 169.405780 1.293935 54.625275 0.000000 0.000000
 81.000000 168.099327 1.297960 54.586798 0.000000 0.000000
 82.000000 167.218145 1.309217 57.965170 0.000000 0.000000
 83.000000 167.581260 1.283837 54.577191 0.000000 0.000000
 84.000000 166.623740 1.306571 57.589068 0.000000 0.000000
 85.000000 165.348003 1.321213 62.091391 0.000000 0.000000
 86.000000 166.932963 1.302572 56.666949 0.000000 0.000000
 87.000000 167.639644 1.298104 56.869102 0.000000 0.000000
 88.000000 166.694779 1.298531 57.338335 0.000000 0.000000
 89.000000 166.982553 1.309031 58.438967 0.000000 0.000000
 90.000000 165.811008 1.324469 62.603929 0.000000 0.000000
 91.000000 167.809600 1.322790 59.433003 0.000000 0.000000
 92.000000 165.840413 1.324544 63.118703 0.000000 0.000000
 93.000000 168.135998 1.322193 58.001611 0.000000 0.000000
 94.000000 168.410420 1.304937 57.022057 0.000000 0.000000
 95.000000 168.742285 1.329691 58.674056 0.000000 0.000000
 96.000000 167.297732 1.323341 59.588083 0.000000 0.000000
 97.000000 170.612459 1.300567 55.508999 0.000000 0.000000
 98.000000 169.600898 1.313649 55.887768 0.000000 0.000000
 99.000000 167.852009 1.325066 60.075345 0.000000 0.000000
 100.000000 168.977015 1.313866 56.753442 0.000000 0.000000
 101.000000 171.301200 1.311687 57.474750 0.000000 0.000000
 102.000000 169.242108 1.312563 57.113650 0.000000 0.000000
 103.000000 169.408980 1.311800 57.676176 0.000000 0.000000
 104.000000 168.831306 1.305677 57.152521 0.000000 0.000000
 105.000000 168.308368 1.311816 58.575706 0.000000 0.000000
 106.000000 166.798411 1.293681 57.504740 0.000000 0.000000
 107.000000 167.133258 1.309527 60.140689 0.000000 0.000000
 108.000000 171.177092 1.296669 57.330441 0.000000 0.000000
 109.000000 168.754327 1.304570 58.040704 0.000000 0.000000
 110.000000 166.540571 1.308817 59.735830 0.000000 0.000000
 111.000000 167.323657 1.296950 57.694432 0.000000 0.000000
 112.000000 168.109282 1.297783 56.869097 0.000000 0.000000
 113.000000 167.866037 1.305509 59.000709 0.000000 0.000000
 114.000000 167.822481 1.312690 60.780986 0.000000 0.000000
 115.000000 168.098006 1.305575 58.650137 0.000000 0.000000
 116.000000 165.232293 1.320140 63.091635 0.000000 0.000000
 117.000000 168.197958 1.309995 60.225525 0.000000 0.000000
 118.000000 167.629129 1.307707 59.873589 0.000000 0.000000
 119.000000 169.156264 1.315988 58.339788 0.000000 0.000000
 120.000000 166.223819 1.330255 62.607129 0.000000 0.000000
 121.000000 166.933057 1.318380 60.893074 0.000000 0.000000
 122.000000 165.908108 1.311810 61.863067 0.000000 0.000000
 123.000000 165.247425 1.316921 63.395042 0.000000 0.000000
 124.000000 165.591635 1.306370 62.538614 0.000000 0.000000
 125.000000 166.474919 1.320128 61.815842 0.000000 0.000000
 126.000000 165.810630 1.301315 62.383129 0.000000 0.000000
 127.000000 166.803348 1.308738 60.662998 0.000000 0.000000
 128.000000 164.910019 1.313608 63.559688 0.000000 0.000000
 129.000000 169.591175 1.295172 55.649012 0.000000 0.000000
 130.000000 167.764429 1.322375 63.001188 0.000000 0.000000
 131.000000 166.889207 1.317966 62.170984 0.000000 0.000000
 132.000000 168.463996 1.295659 57.826188 0.000000 0.000000
 133.000000 167.490694 1.296934 58.354687 0.000000 0.000000
 134.000000 168.368879 1.297284 59.197690 0.000000 0.000000
 135.000000 166.056989 1.300188 60.419808 0.000000 0.000000
 136.000000 167.169243 1.304646 61.222320 0.000000 0.000000
 137.000000 166.157937 1.293021 60.160165 0.000000 0.000000
 138.000000 168.469213 1.301186 59.713555 0.000000 0.000000
 139.000000 166.458546 1.305944 61.247611 0.000000 0.000000
 140.000000 167.834851 1.292572 60.620289 0.000000 0.000000
 141.000000 169.328885 1.305845 58.571524 0.000000 0.000000
 142.000000 167.948641 1.283729 58.719334 0.000000 0.000000
 143.000000 170.112081 1.292433 55.356828 0.000000 0.000000
 144.000000 167.462706 1.302172 59.531878 0.000000 0.000000
 145.000000 168.589309 1.296875 59.240398 0.000000 0.000000
 146.000000 170.792876 1.280671 54.558993 0.000000 0.000000
 147.000000 165.381327 1.294802 61.097202 0.000000 0.000000
 148.000000 167.318964 1.299401 60.232351 0.000000 0.000000
 149.000000 166.759817 1.292910 60.367291 0.000000 0.000000
 150.000000 165.771918 1.291743 62.344634 0.000000 0.000000
 151.000000 167.152173 1.295957 61.602197 0.000000 0.000000
 152.000000 169.761065 1.272163 55.586939 0.000000 0.000000
 153.000000 169.947458 1.277137 57.536402 0.000000 0.000000
 154.000000 167.741116 1.300603 62.769039 0.000000 0.000000
 155.000000 172.924133 1.276169 53.222981 0.000000 0.000000
 156.000000 168.936000 1.291415 56.935678 0.000000 0.000000
 157.000000 171.399835 1.296547 57.652106 0.000000 0.000000
 158.000000 168.218023 1.286365 59.701644 0.000000 0.000000
 159.000000 167.128077 1.290756 61.703065 0.000000 0.000000
 160.000000 170.859434 1.282832 55.825035 0.000000 0.000000
 161.000000 169.455701 1.304444 58.827095 0.000000 0.000000
 162.000000 172.125450 1.281644 54.332544 0.000000 0.000000
 163.000000 173.894356 1.276366 51.620129 0.000000 0.000000
 164.000000 171.930686 1.280055 54.623028 0.000000 0.000000
 165.000000 175.832126 1.273697 49.037545 0.000000 0.000000
 166.000000 170.256386 1.306532 56.367920 0.000000 0.000000
 167.000000 171.263477 1.286035 57.778379 0.000000 0.000000
 168.000000 172.720447 1.277548 51.421564 0.000000 0.000000
 169.000000 171.136360 1.275849 57.549768 0.000000 0.000000
 170.000000 172.044451 1.281695 55.798878 0.000000 0.000000
 171.000000 172.980188 1.284642 55.093688 0.000000 0.000000
 172.000000 171.712316 1.299988 55.606059 0.000000 0.000000
 173.000000 171.505203 1.307021 58.558356 0.000000 0.000000
 174.000000 172.151206 1.315688 57.553824 0.000000 0.000000
 175.000000 172.594056 1.310428 54.116191 0.000000 0.000000
 176.000000 170.900152 1.297102 58.173517 0.000000 0.000000
 177.000000 171.882814 1.282798 56.248040 0.000000 0.000000
 178.000000 167.683669 1.315027 64.348090 0.000000 0.000000
 179.000000 173.212888 1.300337 56.277566 0.000000 0.000000
 180.000000 171.405576 1.299572 59.912856 0.000000 0.000000
 181.000000 173.184931 1.284890 56.665161 0.000000 0.000000
 182.000000 172.554227 1.284978 54.602458 0.000000 0.000000
 183.000000 172.713086 1.284452 53.648661 0.000000 0.000000
 184.000000 170.470546 1.292353 56.218190 0.000000 0.000000
 185.000000 170.606858 1.296587 57.263286 0.000000 0.000000
 186.000000 174.083418 1.289114 51.958834 0.000000 0.000000
 187.000000 174.599223 1.292422 53.186035 0.000000 0.000000
 188.000000 174.015962 1.291293 52.980391 0.000000 0.000000
 189.000000 172.937018 1.288143 56.373131 0.000000 0.000000
 190.000000 174.087120 1.277678 53.019854 0.000000 0.000000
 191.000000 172.035812 1.288896 58.099104 0.000000 0.000000
 192.000000 171.695093 1.301507 57.380637 0.000000 0.000000
 193.000000 172.896576 1.283410 56.557995 0.000000 0.000000
 194.000000 171.341711 1.308204 61.397853 0.000000 0.000000
 195.000000 173.445445 1.287378 55.116989 0.000000 0.000000
 196.000000 174.016903 1.298678 54.725142 0.000000 0.000000
 197.000000 172.575051 1.315472 56.765645 0.000000 0.000000
 198.000000 174.973792 1.301265 53.827175 0.000000 0.000000
 199.000000 172.709699 1.301184 56.107793 0.000000 0.000000
 200.000000 174.702924 1.288087 52.730707 0.000000 0.000000
 201.000000 172.426653 1.289154 57.385960 0.000000 0.000000
 202.000000 170.665316 1.292239 57.703709 0.000000 0.000000
 203.000000 173.586497 1.283929 53.061161 0.000000 0.000000
 204.000000 169.046764 1.290322 57.388117 0.000000 0.000000
 205.000000 170.230272 1.306353 57.448921 0.000000 0.000000
 206.000000 170.102085 1.289718 57.577524 0.000000 0.000000
 207.000000 169.251853 1.292201 58.645991 0.000000 0.000000
 208.000000 169.574228 1.311467 59.036781 0.000000 0.000000
 209.000000 170.822870 1.284596 58.031304 0.000000 0.000000
 210.000000 171.781542 1.288007 57.418075 0.000000 0.000000
 211.000000 174.269103 1.268442 53.091906 0.000000 0.000000
 212.000000 173.417232 1.288904 56.720633 0.000000 0.000000
 213.000000 174.706691 1.266449 52.615365 0.000000 0.000000
 214.000000 173.249060 1.283726 57.884943 0.000000 0.000000
 215.000000 173.716906 1.261791 52.086508 0.000000 0.000000
 216.000000 171.762950 1.270594 57.045241 0.000000 0.000000
 217.000000 173.765308 1.283883 54.187985 0.000000 0.000000
 218.000000 173.695714 1.280291 54.378261 0.000000 0.000000
 219.000000 174.796777 1.267578 53.289974 0.000000 0.000000
 220.000000 169.601204 1.288110 57.685644 0.000000 0.000000
 221.000000 170.726706 1.283565 58.443785 0.000000 0.000000
 222.000000 170.280375 1.285255 57.100415 0.000000 0.000000
 223.000000 172.871337 1.286324 57.731516 0.000000 0.000000
 224.000000 172.306285 1.277670 56.600402 0.000000 0.000000
 225.000000 170.230054 1.279645 57.437636 0.000000 0.000000
 226.000000 173.475088 1.266615 56.648852 0.000000 0.000000
 227.000000 176.281358 1.284503 50.044010 0.000000 0.000000
 228.000000 172.170431 1.293633 58.582149 0.000000 0.000000
 229.000000 173.022124 1.294884 59.385219 0.000000 0.000000
 230.000000 172.647495 1.287231 57.127998 0.000000 0.000000
 231.000000 173.678814 1.287429 55.710634 0.000000 0.000000
 232.000000 175.825557 1.278991 50.137309 0.000000 0.000000
 233.000000 175.960241 1.272077 49.256235 0.000000 0.000000
 234.000000 175.327280 1.285004 52.678227 0.000000 0.000000
 235.000000 175.020956 1.279731 54.497574 0.000000 0.000000
 236.000000 177.353477 1.271647 45.492769 0.000000 0.000000
 237.000000 175.054987 1.296631 55.580857 0.000000 0.000000
 238.000000 177.003978 1.285359 49.278713 0.000000 0.000000
 239.000000 176.326357 1.279946 50.060874 0.000000 0.000000
 240.000000 176.897695 1.293513 50.040926 0.000000 0.000000
 241.000000 174.688018 1.288492 55.155234 0.000000 0.000000
 242.000000 178.523837 1.280421 44.305084 0.000000 0.000000
 243.000000 177.901544 1.286232 46.523769 0.000000 0.000000
 244.000000 179.718761 1.299606 41.045294 0.000000 0.000000
 245.000000 178.256995 1.294457 44.322344 0.000000 0.000000
 246.000000 175.058349 1.305258 56.586277 0.000000 0.000000
 247.000000 179.806912 1.284193 41.331450 0.000000 0.000000
 248.000000 180.643676 1.278080 38.754855 0.000000 0.000000
 249.000000 176.588117 1.316891 49.780916 0.000000 0.000000
 250.000000 177.607751 1.290127 46.757369 0.000000 0.000000
 251.000000 176.530077 1.310702 50.408693 0.000000 0.000000
 252.000000 179.706157 1.290333 42.242446 0.000000 0.000000
 253.000000 177.906465 1.292997 47.719676 0.000000 0.000000
 254.000000 181.869459 1.269256 31.355223 0.000000 0.000000
 255.000000 181.449529 1.271547 33.679515 0.000000 0.000000
 256.000000 179.069057 1.281490 42.044412 0.000000 0.000000
 257.000000 181.759324 1.285478 33.229399 0.000000 0.000000
 258.000000 180.722054 1.288138 39.776968 0.000000 0.000000
 259.000000 179.731103 1.276708 42.663386 0.000000 0.000000
 260.000000 179.110073 1.271676 42.159595 0.000000 0.000000
 261.000000 178.870715 1.276220 44.093383 0.000000 0.000000
 262.000000 178.824822 1.282382 45.546928 0.000000 0.000000
 263.000000 177.641118 1.290482 47.807063 0.000000 0.000000
 264.000000 178.237070 1.292604 45.121936 0.000000 0.000000
 265.000000 177.523879 1.284427 47.949332 0.000000 0.000000
 266.000000 178.605203 1.285080 46.434829 0.000000 0.000000
 267.000000 181.793073 1.282046 33.938274 0.000000 0.000000
 268.000000 182.986227 1.271614 28.760389 0.000000 0.000000
 269.000000 181.549682 1.280890 34.786347 0.000000 0.000000
 270.000000 180.357630 1.273666 40.497994 0.000000 0.000000
 271.000000 182.753702 1.273330 28.882086 0.000000 0.000000
 272.000000 181.670835 1.264220 34.827706 0.000000 0.000000
 273.000000 179.473835 1.287246 41.888995 0.000000 0.000000
 274.000000 179.233618 1.267119 41.856006 0.000000 0.000000
 275.000000 180.111153 1.272964 40.059208 0.000000 0.000000
 276.000000 179.870862 1.286399 43.118631 0.000000 0.000000
 277.000000 180.895045 1.288555 40.679048 0.000000 0.000000
 278.000000 177.955314 1.292185 48.045940 0.000000 0.000000
 279.000000 178.183765 1.303834 46.460614 0.000000 0.000000
 280.000000 184.905568 1.260658 20.573757 0.000000 0.000000
 281.000000 183.392820 1.269155 25.365729 0.000000 0.000000
 282.000000 185.065966 1.262194 18.246911 0.000000 0.000000
 283.000000 186.021535 1.266561 11.296778 0.000000 0.000000
 284.000000 186.428710 1.272822 11.164834 0.000000 0.000000
 285.000000 185.747187 1.269968 11.208138 0.000000 0.000000
 286.000000 183.942419 1.289130 23.747449 0.000000 0.000000
 287.000000 181.934217 1.282148 33.987176 0.000000 0.000000
 288.000000 185.602281 1.275342 13.842499 0.000000 0.000000
 289.000000 183.480220 1.278913 26.458953 0.000000 0.000000
 290.000000 182.491244 1.271159 31.305582 0.000000 0.000000
 291.000000 182.022615 1.282114 33.735782 0.000000 0.000000
 292.000000 181.426042 1.281577 36.360000 0.000000 0.000000
 293.000000 179.192813 1.285092 43.843687 0.000000 0.000000
 294.000000 179.873240 1.286068 43.932407 0.000000 0.000000
 295.000000 183.761081 1.275357 23.864647 0.000000 0.000000
 296.000000 180.266507 1.282755 42.218478 0.000000 0.000000
 297.000000 176.115310 1.299618 52.088051 0.000000 0.000000
 298.000000 176.501399 1.281448 50.279294 0.000000 0.000000
 299.000000 177.164374 1.298557 50.549261 0.000000 0.000000
 300.000000 174.461104 1.310026 56.773256 0.000000 0.000000
 301.000000 173.965365 1.305535 56.582462 0.000000 0.000000
 302.000000 178.144117 1.288631 46.866376 0.000000 0.000000
 303.000000 175.706924 1.301898 51.921113 0.000000 0.000000
 304.000000 179.267837 1.278454 43.757448 0.000000 0.000000
 305.000000 175.718914 1.287356 52.434167 0.000000 0.000000
 306.000000 174.959043 1.310225 57.712048 0.000000 0.000000
 307.000000 176.438326 1.301441 51.973211 0.000000 0.000000
 308.000000 175.558409 1.291310 51.307881 0.000000 0.000000
 309.000000 178.428758 1.278719 46.077114 0.000000 0.000000
 310.000000 179.685863 1.268588 44.019904 0.000000 0.000000
 311.000000 180.449540 1.270119 41.592549 0.000000 0.000000
 312.000000 178.720988 1.270741 47.113839 0.000000 0.000000
 313.000000 178.979414 1.287712 46.294255 0.000000 0.000000
 314.000000 178.391049 1.261238 46.317637 0.000000 0.000000
 315.000000 178.544620 1.264102 47.462008 0.000000 0.000000
 316.000000 178.657009 1.260334 48.405413 0.000000 0.000000
 317.000000 183.577671 1.237478 24.608896 0.000000 0.000000
 318.000000 184.031423 1.233216 21.674494 0.000000 0.000000
 319.000000 184.124810 1.241666 22.165916 0.000000 0.000000
 320.000000 184.432402 1.242099 20.218351 0.000000 0.000000
 321.000000 186.391952 1.241591 10.569142 0.000000 0.000000
 322.000000 185.004976 1.251118 20.294110 0.000000 0.000000
 323.000000 185.250890 1.254026 15.369151 0.000000 0.000000
 324.000000 185.376949 1.258900 15.199914 0.000000 0.000000
 325.000000 184.156545 1.250441 23.474239 0.000000 0.000000
 326.000000 181.195411 1.288636 38.581527 0.000000 0.000000
 327.000000 182.662782 1.293341 31.401137 0.000000 0.000000
 328.000000 184.304364 1.266259 23.870419 0.000000 0.000000
 329.000000 180.638563 1.276894 41.183231 0.000000 0.000000
 330.000000 182.051682 1.263735 33.637443 0.000000 0.000000
 331.000000 183.024739 1.270863 29.942039 0.000000 0.000000
 332.000000 182.618358 1.278246 32.209366 0.000000 0.000000
 333.000000 185.841171 1.255326 11.868271 0.000000 0.000000
 334.000000 184.140283 1.279922 26.685366 0.000000 0.000000
 335.000000 182.028641 1.266177 34.844738 0.000000 0.000000
 336.000000 182.823515 1.251561 30.385578 0.000000 0.000000
 337.000000 182.153291 1.270521 35.271288 0.000000 0.000000
 338.000000 183.671966 1.253024 25.195326 0.000000 0.000000
 339.000000 185.863055 1.238330 12.172962 0.000000 0.000000
 340.000000 185.604647 1.244583 14.381998 0.000000 0.000000
 341.000000 186.280895 1.250501 11.407510 0.000000 0.000000
 342.000000 183.266366 1.259190 27.740678 0.000000 0.000000
 343.000000 183.729218 1.262741 26.109047 0.000000 0.000000
 344.000000 186.811075 1.247608 8.101884 0.000000 0.000000
 345.000000 185.540878 1.267519 16.917079 0.000000 0.000000
 346.000000 186.492077 1.271987 13.487947 0.000000 0.000000
 347.000000 188.071344 1.261161 11.103216 0.000000 0.000000
 348.000000 186.272298 1.272658 14.023152 0.000000 0.000000
 349.000000 187.047637 1.267278 10.314277 0.000000 0.000000
 350.000000 185.050395 1.267316 21.508940 0.000000 0.000000
 351.000000 185.647968 1.271805 17.334968 0.000000 0.000000
 352.000000 187.721258 1.260834 10.361204 0.000000 0.000000
 353.000000 185.398274 1.261268 17.746305 0.000000 0.000000
 354.000000 184.664084 1.270534 20.517695 0.000000 0.000000
 355.000000 186.679129 1.261832 10.991109 0.000000 0.000000
 356.000000 185.231934 1.266843 19.417073 0.000000 0.000000
 357.000000 183.837071 1.278478 27.371932 0.000000 0.000000
 358.000000 183.813487 1.293013 29.091795 0.000000 0.000000
 359.000000 182.686395 1.271974 32.844043 0.000000 0.000000
 360.000000 185.792321 1.278899 16.739903 0.000000 0.000000
 361.000000 186.597091 1.263601 13.410695 0.000000 0.000000
 362.000000 183.819739 1.294397 30.122299 0.000000 0.000000
 363.000000 185.955843 1.286872 16.620583 0.000000 0.000000
 364.000000 186.177071 1.274012 14.839474 0.000000 0.000000
 365.000000 185.872632 1.268124 17.510974 0.000000 0.000000
 366.000000 184.674288 1.263952 21.290227 0.000000 0.000000
 367.000000 185.160497 1.263128 21.151774 0.000000 0.000000
 368.000000 184.446133 1.256803 23.479028 0.000000 0.000000
 369.000000 181.240095 1.276051 38.759100 0.000000 0.000000
 370.000000 183.555115 1.264204 29.290688 0.000000 0.000000
 371.000000 183.606978 1.268470 30.528491 0.000000 0.000000
 372.000000 186.176315 1.246939 14.680969 0.000000 0.000000
 373.000000 181.973207 1.256309 36.170254 0.000000 0.000000
 374.000000 181.793392 1.276781 37.469447 0.000000 0.000000
 375.000000 180.013878 1.268935 41.875969 0.000000 0.000000
 376.000000 180.657608 1.266920 41.486028 0.000000 0.000000
 377.000000 179.364018 1.284047 44.152651 0.000000 0.000000
 378.000000 183.425348 1.260375 29.600384 0.000000 0.000000
 379.000000 181.816826 1.267136 38.087493 0.000000 0.000000
 380.000000 183.257515 1.256369 29.403038 0.000000 0.000000
