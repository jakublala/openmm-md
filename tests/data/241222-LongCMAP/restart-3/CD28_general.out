#"Progress (%)"	"Step"	"Time (ps)"	"Potential Energy (kJ/mole)"	"Kinetic Energy (kJ/mole)"	"Total Energy (kJ/mole)"	"Temperature (K)"	"Box Volume (nm^3)"	"Density (g/mL)"	"Speed (ns/day)"	"Time Remaining"
0.0%	50000	147099.99984894163	-4387968.645964377	674914.6942853052	-3713053.9516790714	299.9773825277815	2744.0	0.9828403395655725	0	--
0.0%	100000	147199.9998493258	-4391327.784080397	674474.8909536793	-3716852.893126718	299.78190441274677	2744.0	0.9828403395655725	28.3	35:6:30:15
0.0%	150000	147299.99984970997	-4392390.769238345	678100.0401032655	-3714290.72913508	301.3931639724712	2744.0	0.9828403395655725	28.3	35:7:00:13
